java/core/src/main/java/co/worklytics/psoxy/gateway/impl/CommonRequestHandler.java

```diff
@@ -411,7 +411,9 @@ private com.google.api.client.http.HttpHeaders getHeadersFromSource(HttpEventReq
 
         sanitizer.getSupportedHeaders(request.getHttpMethod(), targetUrl)
                 .ifPresent(i -> i.forEach(h -> {
-                    request.getHeader(h).ifPresent(headerValue -> headers.set(h, headerValue));
+                    request.getHeader(h).ifPresent(headerValue -> {
+                        log.info(String.format("Header %s included", h));
```

Comment:
may be too verbose for prod use, although i understand that in practice we don't have very many cases where we're forwarding headers to source